docs/config.md

```diff
@@ -130,9 +121,18 @@ OpenTelemetry .NET Automatic Instrumentation.
 
 To perform bytecode instrumentation, configure the OpenTelemetry .NET
 Automatic Instrumentation as a .NET CLR Profiler. The CLR uses the following
-environment variables to set up the profiler. 
+environment variables to set up the profiler. See [.NET Runtime Profiler Loading](https://github.com/dotnet/runtime/blob/main/docs/design/coreclr/profiling/Profiler%20Loading.md) for more information.
+
+### .NET Framework
+| Environment variable | Description | Value |
+|-|-|-|
+| `COR_ENABLE_PROFILING` | Enables the profiler. | `1` |
+| `COR_PROFILER` | CLSID of the profiler. | `30000` (ms) |
+| `COR_PROFILER_PATH` | Path to the profiler. | `%InstallationLocation%/OpenTelemetry.AutoInstrumentation.Native.dll` |
+| `COR_PROFILER_PATH_32` | Path to the 32-bit profiler. Bitness-specific paths take precedence over generic paths. | `%InstallationLocation%/win-x86/OpenTelemetry.AutoInstrumentation.Native.dll` |
+| `COR_PROFILER_PATH_64` | Path to the 64-bit profiler. Bitness-specific paths take precedence over generic paths. | `%InstallationLocation%/win-x64/OpenTelemetry.AutoInstrumentation.Native.dll` |
 
-> Notice that .NET Framework uses the `COR_` prefix instead of `CORECLR_`.
```

Comment:
Can we keep a single table and perhaps call more attention to the different prefixes?

Reply:
Tried to address this in a5c96d92f0460ff7a67b90cc3846a6671024ce80, let me know what you think